### PR TITLE
Update tendermint-rpc README

### LINKED ISSUE #1034: Review @iov/tendermint-rpc README
- Format text
- Remove reference to old version v0.25
- Update all the rest

### PR DESCRIPTION
**Pull request by webmaster128**
Closes #1034

### CODE REVIEW

File: packages/iov-tendermint-rpc/README.md [hunk 1]
```diff
@@ -2,58 +2,81 @@
...
-The actual domain knowledge is embeded in the [Adaptor](https://iov-one.github.io/iov-core-docs/latest/iov-tendermint-rpc/modules/_adaptor_.html),
-which defines a class for encoding [Params](https://iov-one.github.io/iov-core-docs/latest/iov-tendermint-rpc/interfaces/_adaptor_.params.html)
-and another for decoding [Responses](https://iov-one.github.io/iov-core-docs/latest/iov-tendermint-rpc/interfaces/_adaptor_.responses.html).
-There is currently an [implementation for v0.25](https://iov-one.github.io/iov-core-docs/latest/iov-tendermint-rpc/modules/_v0_25_index_.html). This knowledge is mainly for those who
-want to add support for new versions, which should be added to the
-[auto-detect method](https://iov-one.github.io/iov-core-docs/latest/iov-tendermint-rpc/classes/_client_.client.html#detectversion).
+The main entry point is the
+[Client](https://iov-one.github.io/iov-core-docs/latest/iov-tendermint-rpc/classes/client.html).
+
+The connection to the blockchain is defned by a flexible
```
Comment (willclarktech) [inline packages/iov-tendermint-rpc/README.md]: "defined"

File: packages/iov-tendermint-rpc/README.md [hunk 2]
```diff
@@ -2,58 +2,81 @@
...
+[RpcClient](https://iov-one.github.io/iov-core-docs/latest/iov-tendermint-rpc/interfaces/rpcclient.html)
+interface, with implmentations for Http (POST) and Websockets. You can add your
```
Comment (willclarktech) [inline packages/iov-tendermint-rpc/README.md]: "implementations for HTTP"

File: packages/iov-tendermint-rpc/README.md [hunk 3]
```diff
@@ -2,58 +2,81 @@
...
+own connection type or just wrap one with custom retry rules, error handling,
+etc. The RPC client is just responsible for sending JSON-RPC requests and
+returning the responses.
+
+The actual domain knowledge is embeded in the
```
Comment (willclarktech) [inline packages/iov-tendermint-rpc/README.md]: "embedded"

File: packages/iov-tendermint-rpc/README.md [hunk 4]
```diff
@@ -2,58 +2,81 @@
...
+[Adaptor](https://iov-one.github.io/iov-core-docs/latest/iov-tendermint-rpc/interfaces/adaptor.html),
+which defines a class for encoding
+[Params](https://iov-one.github.io/iov-core-docs/latest/iov-tendermint-rpc/classes/params.html)
+and another for decoding
+[Responses](https://iov-one.github.io/iov-core-docs/latest/iov-tendermint-rpc/classes/responses.html).
+The Tendermint version specific functionality is implemented in global objects
```
Comment (willclarktech) [inline packages/iov-tendermint-rpc/README.md]: "version-specific"

**Comment by webmaster128** [comment]
Updated and one new commit added.

**Pull request merged**